### ISSUE TITLE
cp, cpr, cpy classes

### ISSUE THREAD

**Issue by jncraig** [nvisionative/nvQuickTheme#149]
## Please summarize your question in one sentence

What are these classes, used in default.aspx to define pane wrappers

**Comment by nvisionative** [comment]
@jncraig we'll assume you meant the `default.ascx` file.  These classes are implemented in [this file](https://github.com/nvisionative/nvQuickTheme/blob/69097053b7c04fadb700cd287b01ef0486c566e4/src/scss/components/_helpers.scss) and it is explained at a high level there.  However, in layman terms:

- cp = content padding
- y = y axis
- x = x axis
- t = top
- b = bottom
- l = left
- r = right
- # = size

So, `cpy-4` would add content padding of to the top and bottom.  `cpx-4` to the left and right.  `cpl-4` just to the left, `cpr-4` to the right, etc.

**Comment by jncraig** [comment]
Thanks.  I eventually found the other paddings, too.
